tb_plugin/torch_tb_profiler/profiler/trace.py

```diff
@@ -9,7 +9,7 @@
 
 
 class EventTypes(object):
-    NET = "Net"
```

Comment:
would this change ignore all events whose category is Net?

Reply:
The plugin has no use of NET.